tests/ttnn/integration_tests/stable_diffusion/test_cross_attn_up_block_2d_new_conv.py

```diff
@@ -215,7 +215,7 @@ def test_cross_attn_up_block_2d_512x512(
     out_channels,
 ):
     # TODO
-    # pytest.skip()
+    pytest.skip()
```

Comment:
Also, can you please do `@pytest.mark.skip(reason="...")` at the top of the test function instead as a decorator?

Reply:
Done